modules/tinymce/src/plugins/importcss/main/ts/core/ImportCss.ts

```diff
@@ -33,6 +33,8 @@ interface Group extends UserDefinedGroup {
   readonly filter: Filter | undefined;
 }
 
+const INTERNAL_EDITOR_STYLE = /^\.(?:ephox|tiny|mce)(?:-+\w+)+$/;
```

Comment:
I don't think I've seen many all caps snake case constants, is this common?

Reply:
I swear that used to be the style... I guess I should've checked. Two years away from the code and I forget some things.

Reply:
I don't see very many module-level constants but those I have found are camel case not upper-snake case. I'll change it.

Reply:
We've never used upper snake case since I've been here fwiw. I remember doing this when I first started and got pinged because it wasn't our normal style 🤷

Reply:
yeah I definitely forget these things after so much time away, and having used upper-snake case on previous codebases 😂